### DISCUSSION TITLE
Login for `project up`?

### DISCUSSION THREAD

**Discussion by SteinRobert** [unikubehq/cli#72] [Ideas]
I was thinking whether it makes sense to force the login on `unikube project up` or not?

I've been stumbling over this quite a few time. Feels a bit humped - but maybe it's just me. Wouldn't it make sense to limit the commands which actually need a login?

**Comment by Schille** [comment]
I extended the token lifetime for CLI client to 5 days for now. But anyway, all destructive actions such as `project down` or `... delete` should be executable without a valid login statis @buschNT

**Reply by buschNT** [reply]
See issue #102 .
